.mocharc.yaml

```diff
@@ -0,0 +1,10 @@
+exit: true
```

Comment:
Let's make this json :)

Reply:
Not all of the configuration files use JSON, e.g. Travis configuration is YAML.

Yaml has this advantage over JSON, that it supports comments, which is quite important when you want to have everything well documented. AFAIK Mocha does not support JSON5, so the most reasonable format that supports comments seems to be yaml.

Given the above, can we stick to YAML?

Reply:
Okay, sure that sounds reasonable.